src/Lean/Elab/AutoBound.lean

```diff
@@ -24,7 +26,11 @@ register_builtin_option relaxedAutoImplicit : Bool := {
     descr    := "When \"relaxed\" mode is enabled, any atomic nonempty identifier is eligible for auto bound implicit locals (see option `autoImplicit`)."
   }
 
-
+/--
+Intuitively a variable can be auto-bound in strict mode if it is a single character (`α` or `x`),
```

Comment:
I don't think this is strictly speaking intuitive.

Normally we also use the indicative mood for these kinds of things:
```suggestion
Checks whether a string is a name that could be automatically implicitly bound when
in strict mode.

A variable can be auto-bound in strict mode if it is a single character (`α` or `x`),
```